admiral/tools/utils.py

```diff
@@ -28,8 +33,11 @@ def checkpoint_from_trained_directory(full_trained_directory, checkpoint_desired
             if checkpoint_desired == int(checkpoint.split('/')[-1].split('_')[-1]):
                 return checkpoint, checkpoint_desired
         import warnings
```

Comment:
Actually flake won't catch this because it is in a function, which is a good python practice. This is a function-level import, not a module-level import